openreview/venue_request/venue_request.py

```diff
@@ -1134,11 +1134,21 @@ def setup_request_form(self):
                 'required': True,
                 'order': 6
             },
+            'publication_chairs': {
+                'description': 'Will your venue have Publication Chairs? The Publication Chairs will only have access to accepted submissions (including author names) and the author accepted group in order to email authors of accepted submissions.',
+                'value-radio': [
+                    'Yes, our venue has Publication Chairs',
+                    'No, our venue does not have Publication Chairs'
+                ],
+                'required': False,
```

Comment:
can you set this field to be required true?

Reply:
Yep, wanted to make sure it would not break anything on the live site. I need to fix all the tests that use the request form.